src/main/java/com/bloxbean/cardano/client/util/PolicyUtil.java

```diff
@@ -0,0 +1,60 @@
+package com.bloxbean.cardano.client.util;
+
+import com.bloxbean.cardano.client.crypto.KeyGenUtil;
+import com.bloxbean.cardano.client.crypto.Keys;
+import com.bloxbean.cardano.client.crypto.SecretKey;
+import com.bloxbean.cardano.client.crypto.VerificationKey;
+import com.bloxbean.cardano.client.exception.CborSerializationException;
+import com.bloxbean.cardano.client.transaction.spec.Policy;
+import com.bloxbean.cardano.client.transaction.spec.script.RequireTimeBefore;
+import com.bloxbean.cardano.client.transaction.spec.script.ScriptAll;
+import com.bloxbean.cardano.client.transaction.spec.script.ScriptAtLeast;
+import com.bloxbean.cardano.client.transaction.spec.script.ScriptPubkey;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class PolicyUtil {
+
+    private static final int SLOTS_PER_EPOCH = 5 * 24 * 60 * 60;
```

Comment:
@edridudi Should we get this from Genesis info instead of hard coded value ?

[NetworkInfoService.getNetworkInfo()](https://github.com/bloxbean/cardano-client-lib/blob/master/src/main/java/com/bloxbean/cardano/client/backend/api/NetworkInfoService.java#L13)

We can create a separate issue to handle this.

Reply:
Good point. 
It's not crucial because SlotsPerEpoch is not about to change in the new future but yes I will create an issue for that.